assets/js/blocks/filter-wrapper/block.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "woocommerce/filter-wrapper",
+	"version": "1.0.0",
+	"title": "Active Product Filters",
+	"description": "Show the currently active product filters. Works in combination with other filters blocks.",
```

Comment:
it's probably better to register the Active Product Filters as a variation too, but we need to hide the main filter wrapper block.